apps/etterlatte-behandling/src/main/kotlin/behandling/klage/KlageService.kt

```diff
@@ -232,9 +234,22 @@ class KlageServiceImpl(
         val klageMedOppdatertUtfall = klage.oppdaterUtfall(utfallMedBrev)
         klageDao.lagreKlage(klageMedOppdatertUtfall)
 
+        if (skalOppretteVedtak(utfallMedBrev)) {
+            runBlocking {
+                vedtakKlient.lagreVedtakKlage(klageMedOppdatertUtfall, saksbehandler)
+            }
+        }
+
         return klageMedOppdatertUtfall
     }
 
+    private fun skalOppretteVedtak(utfall: KlageUtfallMedData): Boolean {
```

Comment:
Jeg tenker det er interessant å lagre ned vedtaksid'en for klagen som en del av avvist-klage-vedtak-greien. Så jeg ville heller ha kjørt dette fra 

```
is KlageUtfallUtenBrev.Avvist -> 
```
-delen, slik at man kan få det lagret ned med klagen i behandling. 

Det betyr at vedtaket ikke er lagret med siste versjon av klagen, men vil uansett kunne være endringer før fatting, så jeg tenker vi pusher den nyeste klagen over til vedtak når vi fatter / attesterer / returnerer